src/components/v5/common/ActionSidebar/partials/forms/ManagePermissionsForm/consts.ts

```diff
@@ -6,6 +6,16 @@ import { formatText } from '~utils/intl.ts';
 import { ACTION_BASE_VALIDATION_SCHEMA } from '~v5/common/ActionSidebar/consts.ts';
 import { type CardSelectOption } from '~v5/common/Fields/CardSelect/types.ts';
 
+export const FIELD_NAME = {
+  MEMBER: 'member',
+  TEAM: 'team',
+  CREATED_IN: 'createdIn',
+  ROLE: 'role',
+  AUTHORITY: 'authority',
+  PERMISSIONS: 'permissions',
+  DECISION_METHOD: 'decisionMethod',
```

Comment:
Looks like we ended up with 2 sources for the decision method field name, this will only work as long as `DECISION_METHOD_FIELD_NAME` has exactly the same value, which is not obvious.

Please either set the value of this property to the existing `DECISION_METHOD_FIELD_NAME`, or pass `FIELD_NAME.DECISION_METHOD` as a prop to `DecisionMethodField`, so that we don't rely on 2 separate variables holding the same value.

Is there a reason for using `as const` objects as opposed to string-based enums? I understand they solve some problems with numeric enums but otherwise I think enums would be the preference as per #1768.

Reply:
I have changed the `FIELD_NAME.DECISION_METHOD` to reference `DECISION_METHOD_FIELD_NAME`, so we don't end up with multiple sources of truth. Didn't do it from the beginning as I saw many fields being spread across multiple files and was thinking a bigger refactor task might be needed.

As for the `as const` objects, I have no strong opinion for one or the other. 
However, when trying to refactor to an enum, the `setValue( `${FIELD_NAME.PERMISSIONS}.role_${colonyRole}`, userRole.permissions.includes(colonyRole))` triggered an error that boolean can't be assigned to type never, which I'm not sure yet why it appeared.

Reply:
I think that makes it a good exception to the "prefer enums over const object" guideline.

Also came across this https://github.com/react-hook-form/react-hook-form/issues/8898